k8s/demo/common/bsp/processor-with-ingress.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bsp-demo-public-processor
+  namespace: bsp
+  annotations:
+    flux.weave.works/automated: "true"
+spec:
+  releaseName: bsp-demo-public
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan
+    version: 0.1.3
+  values:
+    bulk-scan-processor:
+      java:
+        ingressHost: ""
```

Comment:
so ingress is not needed for this one? filename says "with ingress"

Reply:
we will use azure app management?

Reply:
then there's no need of having separate processor. it might be only me. perhaps it's easier/safer to assign single processor 🤷‍♂ 

but yea. i think then it's ok

Reply:
change the file name